Markdown/Default.sublime-keymap

```diff
@@ -0,0 +1,41 @@
+[
+    // Auto-pair asterisks
+    { "keys": ["*"], "command": "insert_snippet", "args": {"contents": "*$0*"}, "context":
```

Comment:
This will insert two asterisks when wanting to start a list for example. Probably unwanted.

Reply:
Fixed. The second asterisk is only added if there there is some non-whitespace in the line.